src/krait/utils/templates.py

```diff
@@ -2,4 +2,4 @@
 import jinja2
 
 
-env = jinja2.Environment(loader=jinja2.PackageLoader('krait', 'templates'))
+env = jinja2.Environment(loader=jinja2.PackageLoader('krait', 'templates'), autoescape=False)
```

Comment:
unsure about the implications of not auto escaping, though that's more of a note for myself later on to check